great_expectations/datasource/data_connector/configured_asset_sql_data_connector.py

```diff
@@ -38,6 +45,20 @@ class ConfiguredAssetSqlDataConnector(DataConnector):
     A DataConnector that requires explicit listing of SQL tables you want to connect to.
     """
 
+    SPLITTER_METHOD_TO_SORTER_METHOD_MAPPING: Dict[str, Optional[Sorter]] = {
+        "split_on_year": DictionarySorter,
+        "split_on_year_and_month": DictionarySorter,
+        "split_on_year_and_month_and_day": DictionarySorter,
+        "split_on_date_parts": DictionarySorter,
+        "split_on_whole_table": None,
+        "split_on_column_value": LexicographicSorter,
+        "split_on_converted_datetime": LexicographicSorter,
+        "split_on_divided_integer": NumericSorter,
+        "split_on_mod_integer": NumericSorter,
+        "split_on_multi_column_values": LexicographicSorter,
+        "split_on_hashed_column": LexicographicSorter,
+    }
```

Comment:
Should we make these string values enums? 

If we inherit from both `Enum` and `str`, we could ensure equality between strings and their enum counterparts.

Reply:
@cdkini it could be done, but I would prefer to do it in a follow-up PR since these values go across many more files and tests than the scope of this PR.

Reply:
I am doing this as part of this WIP PR: https://github.com/great-expectations/great_expectations/pull/5980